mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -105,6 +105,21 @@ public static bool SetGCAllowSynchronousMajor (bool flag)
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
 		static extern void SendMicrosoftTelemetry_internal (IntPtr payload, ulong portable_hash, ulong unportable_hash);
 
+		[MethodImplAttribute (MethodImplOptions.InternalCall)]
+		static extern void WriteStateToDisk_internal (IntPtr payload, ulong portable_hash, ulong unportable_hash);
+
+		static void
+		WriteStateToDisk (Exception exc)
```

Comment:
I know it is obnoxious, but I think we need to stop using the term "disk".
Disks are rare these days, what with SSDs, and deleting stuff before it escapes cache anyway -- many writes are only to memory.
"File", "FileSystem" or just "WriteState"?

Reply:
This made me laugh. Yeah I can say File.